scalalib/src/JavaModule.scala

```diff
@@ -352,13 +352,15 @@ trait JavaModule
    */
   def javadocOptions: T[Seq[String]] = T { Seq[String]() }
 
+  def docSources: T[Seq[PathRef]] = allSources
```

Comment:
Please add some scaladoc indicating the purpose of this target and the expected return value. A reference to `docResources` would be nice, too.

Here, in `JavaModule`, you implement `docSources` it with _source directories_, whereas in `ScalaModule` you override with with a list of _source files_. Source directories is the better choice.

Reply:
The first point is fixed in https://github.com/com-lihaoyi/mill/pull/1385/commits/ec90f4843d799a8875dc646e87c681d7fd2b059c and https://github.com/com-lihaoyi/mill/pull/1385/commits/521a41575c444f011eb33fb8f6abb57206c352a0.

The second is fixed in https://github.com/com-lihaoyi/mill/pull/1385/commits/e7934292cf2b2bfc781d788e1e3a530d90dc7b46.